controller/validation.go

```diff
@@ -0,0 +1,45 @@
+package controller
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/expenseledger/web-service/service"
+	"github.com/gin-gonic/gin"
+	"golang.org/x/net/context"
+)
+
+// ValidateHeader validate header
+func ValidateHeader() gin.HandlerFunc {
```

Comment:
Why created a function which returns another function? Can we not just use a non-higher-order function?